import_map.json

```diff
@@ -1,25 +1,28 @@
 {
   "imports": {
     "deps": "./src/deps.ts",
-    "std/": "https://deno.land/std@0.189.0/",
+    "std/": "https://deno.land/std@0.192.0/",
     "cliffy/": "https://deno.land/x/cliffy@v0.25.7/",
     "dnt": "https://deno.land/x/dnt@0.36.0/mod.ts",
     "adm-zip": "https://esm.sh/v124/adm-zip@0.5.9",
     "ajv": "https://esm.sh/v124/ajv@8.11.0",
-    "ajv/dist/2019.js": "https://esm.sh/v124/ajv@8.11.0/dist/2019.js",
     "cdktf": "https://esm.sh/v124/cdktf@0.15.5",
+    "cli-spinners": "https://esm.sh/v124/cli-spinners@2.7.0",
     "constructs": "https://esm.sh/v124/constructs@10.2.27",
     "d3-node": "https://esm.sh/v124/*d3-node@3.0.0",
-    "googleapis": "https://esm.sh/v124/*googleapis@118.0.0",
+    "googleapis": "https://esm.sh/v124/googleapis@118.0.0",
     "js-yaml": "https://esm.sh/v124/js-yaml@4.1.0",
+    "listr": "https://esm.sh/v124/listr@0.14.3",
     "log-update": "https://esm.sh/v124/*log-update@5.0.1",
     "node-plantuml": "https://esm.sh/v124/*node-plantuml@0.9.0",
-    "semver": "https://esm.sh/v124/*semver@7.5.3",
-    "which": "https://esm.sh/v124/*which@3.0.1",
+    "rxjs": "https://esm.sh/v124/rxjs@7.5.6",
```

Comment:
I vaguely recall running into some issue using this version of rxjs instead of npm... I can't exactly remember what it was though, it's possible it's been fixed, just wanted to flag just in case 😛

Reply:
I just converted all of these except for one because it just made more sense to me not to have a mix of package managers that seemed kind of random. The `@kubernetes/client-node` dependency made some Deno processes unhappy when I changed it though, so I left that one as is

Reply:
Yeah I agree, I started off with all of these using the esm.sh version and converted to `npm` only when I hit errors - there may not have been an error with `rxjs`, if you were successfully hitting code using it this change is fine

Reply:
I haven't seen an error with that, so I think it's fine